packages/go_router/doc/configuration.md

```diff
@@ -151,4 +151,4 @@ final _router = GoRouter(
 [GoRouter]: https://pub.dev/documentation/go_router/latest/go_router/GoRouter-class.html
 [GoRoute]: https://pub.dev/documentation/go_router/latest/go_router/GoRoute-class.html
 [GoRouterState]: https://pub.dev/documentation/go_router/latest/go_router/GoRouterState-class.html
-[ShellRoute]: https://pub.dev/documentation/go_router/latest/go_router/ShellRoute-class.html
\ No newline at end of file
+[ShellRoute]: https://pub.dev/documentation/go_router/latest/go_router/ShellRoute-class.html
```

Comment:
I can't tell what has changed in this line. is there a hidden magic here?

Reply:
Hi @sdoshi983 can you revert this change if there is indeed nothing change?

Reply:
hey @chunhtai, yes you are correct. There was no change. As suggested by you:
- I have signed the CLA
- I have reverted the change.

Please let me know in case of any other changes